Makefile

```diff
@@ -178,12 +178,14 @@ clean:
 	rm -rf $(OUT_DIR)
 .PHONY: clean
 
-# Build a release of OpenShift for linux/amd64 and the images that depend on it.
+PLATFORM=$(shell bash -c 'source hack/common.sh; os::build::host_platform')
+
+# Build a release of OpenShift for host platform and the images that depend on it.
 #
 # Example:
 #   make release
 release: clean
-	OS_ONLY_BUILD_PLATFORMS="linux/amd64" hack/build-release.sh
+	OS_ONLY_BUILD_PLATFORMS=$(PLATFORM) hack/build-release.sh
 	hack/build-images.sh
```

Comment:
There will be images built on other platforms (it's possible now).  As long as we can ensure that the builds work correctly for the platforms we have today it's ok to make these more flexible.

I'd prefer that all build variables be prefixed with `OS_BUILD_XXX`.  The less generic the variable, the longer `XXX` should be.